Ansible/roles/cloudstack-manager/tasks/centos8.yml

```diff
@@ -69,9 +69,15 @@
 - name: Ensure CA Certs are latest
   dnf: name=ca-certificates state=latest enablerepo=base
 
-- name: install rng-tools to get entropy
+- name: install haveged to get entropy
   dnf: name=haveged state=present
 
+- name: start haveged for entropy
+  service:
+    name: haveged
+    state: started
+    enabled: yes
```

Comment:
Worth cehcking if simply installing this enable/runs the service?

Reply:
as mentioned above